MQ2GMCheck.cpp

```diff
@@ -1093,54 +1094,71 @@ void DrawGMCheckSettingsPanel()
 	{
 		s_settings.m_GMCheckEnabled.Write(GMCheckEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Turn GM alerting on or off");
 
 	bool GMSoundEnabled = s_settings.m_GMSoundEnabled.Read();
 	if (ImGui::Checkbox("Sound Playing Enabled", &GMSoundEnabled))
 	{
 		s_settings.m_GMSoundEnabled.Write(GMSoundEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Toggle playing sounds for GM alerts, be sure to set the GM Enter/Leave/Reminder file names");
 
 	bool GMBeepEnabled = s_settings.m_GMBeepEnabled.Read();
 	if (ImGui::Checkbox("Beep Enabled", &GMBeepEnabled))
 	{
 		s_settings.m_GMBeepEnabled.Write(GMBeepEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Toggle playing beeps for GM alerts");
 
 	bool GMPopupEnabled = s_settings.m_GMPopupEnabled.Read();
 	if (ImGui::Checkbox("Popup Enabled", &GMPopupEnabled))
 	{
 		s_settings.m_GMPopupEnabled.Write(GMPopupEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Toggle showing popup messages for GM alerts");
 
 	bool GMCorpseEnabled = s_settings.m_GMCorpseEnabled.Read();
 	if (ImGui::Checkbox("Include Corpses", &GMCorpseEnabled))
 	{
 		s_settings.m_GMCorpseEnabled.Write(GMCorpseEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Toggle GM alert being ignored if the spawn is a corpse");
 
 	bool GMChatAlertEnabled = s_settings.m_GMChatAlertEnabled.Read();
 	if (ImGui::Checkbox("Alert in MQ Chat", &GMChatAlertEnabled))
 	{
 		s_settings.m_GMChatAlertEnabled.Write(GMChatAlertEnabled ? FlagOptions::On : FlagOptions::Off);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Toggle GM alert being output to the MQ chat window");
 
 	int GMReminderInterval = s_settings.GetReminderInterval();
 	if (ImGui::SliderInt("Reminder Interval", &GMReminderInterval, 0, 600))
 	{
 		s_settings.SetReminderInterval(GMReminderInterval);
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Set GM reminder interval, in seconds, 0 to disable reminders");
 
 	int LeftVolume = GetPrivateProfileInt("Settings", "LeftVolume", -1, INIFileName);
 	if (LeftVolume > 100 || LeftVolume < 0)
 	{
 		LeftVolume = 50;
 		WritePrivateProfileInt("Settings", "LeftVolume", LeftVolume, INIFileName);
 	}
+
 	if (ImGui::SliderInt("Left Volume", &LeftVolume, 0, 100))
 	{
 		WritePrivateProfileInt("Settings", "LeftVolume", LeftVolume, INIFileName);
 		SetupVolumesFromINI();
 	}
+	ImGui::SameLine();
+	mq::imgui::HelpMarker("Set the volume for alert sounds");
```

Comment:
I'd probably mention "for the left speaker" here, but not a big deal.  (Same with right on the one below)

Reply:
Completed suggested change.